data/migrations/20211102125418_courses.js

```diff
@@ -1,60 +1,68 @@
 exports.up = (knex) => {
   return knex.schema
-    .createTable('course_types', function (table) {
-      table.increments('course_type_id');
-      table.text('description').notNullable();
-      table.string('subject').notNullable().unique();
-      table.specificType('prereqs', 'text ARRAY');
+    .createTable('programs', function (table) {
```

Comment:
I think this approach will be easier, and more inline with the stakeholders requests, good job!